server/src/main/java/org/opensearch/search/internal/ContextIndexSearcher.java

```diff
@@ -445,6 +445,10 @@ public DirectoryReader getDirectoryReader() {
         return (DirectoryReader) reader;
     }
 
+    public SearchContext getSearchContext() {
```

Comment:
Please remove, no need to expose the context here

Reply:
Hi @reta, Thanks for the suggestions and it has been addressed now. The intention is from https://github.com/opensearch-project/OpenSearch/pull/8801#discussion_r1272810102.